src/pydata_sphinx_theme/theme/pydata_sphinx_theme/_templates/version-switcher.html

```diff
@@ -72,6 +72,12 @@
             if (entry.version == "{{ theme_switcher.get('version_match') }}") {
                 node.classList.add("active");
                 $("#version_switcher_button").text(entry.name);
+                // Add extra class to the entry
+                if ("extra_classes" in entry) {
+                    $.each(entry.extra_classes, function(index, extra_class) {
+                      node.classList.add(extra_class);
```

Comment:
This is still adding the class to the entry link, I think? (although now only to the active version)

if I understand correctly, the current code will add the class in the html (as it is after being generated with the javascript, simplified a bit for explanation purposes) where I labeled it "HERE" (so in the first `<a>` node):


```html
<div class="dropdown show" id="version_switcher">
  <button type="button" class="..." id="version_switcher_button" ...>
    0.8.0 <span class="caret"></span>
  </button>
  <div id="version_switcher_menu" class="dropdown-menu ..." ...>
    <a class="list-group-item ... HERE" href="https://pydata-sphinx-theme.readthedocs.io/en/v0.7.1/index.html">v0.8.0 (stable)</a>
    <a class="list-group-item ..." href="https://pydata-sphinx-theme.readthedocs.io/en/v0.7.1/index.html">v0.7.1</a>
    <a class="list-group-item ..." href="https://pydata-sphinx-theme.readthedocs.io/en/v0.7.0/index.html" >0.7.0</a >
    ...
  </div>
</div>
```

While my understanding is that we want to be able to color the dropdown button, so have it instead at this location:

```html
<div class="dropdown show" id="version_switcher">
  <button type="button" class="... HERE" id="version_switcher_button" ...>
    0.8.0 <span class="caret"></span>
  </button>
  <div id="version_switcher_menu" class="dropdown-menu ..." ...>
    <a class="list-group-item ..." href="https://pydata-sphinx-theme.readthedocs.io/en/v0.7.1/index.html">v0.8.0 (stable)</a>
    <a class="list-group-item ..." href="https://pydata-sphinx-theme.readthedocs.io/en/v0.7.1/index.html">v0.7.1</a>
    <a class="list-group-item ..." href="https://pydata-sphinx-theme.readthedocs.io/en/v0.7.0/index.html" >0.7.0</a >
    ...
  </div>
</div>
```

Reply:
ah ok I see. I move the code to `version_switcher_button`.

Reply:
The changes are in. Can you have another look @jorisvandenbossche?